resources/sass/modules/_introduction.scss

```diff
@@ -0,0 +1,69 @@
+.introSlide {
```

Comment:
very minor and i hope you see this. If you could put all these classes in a `template` folder under `modules` that would help too. So it'd be `resources/sass/modules/templates/_introductions.scss`